src/towncrier/_builder.py

```diff
@@ -129,10 +142,20 @@ def find_fragments(
         file_content = {}
 
         for basename in files:
+            # Skip files that are deliberately ignored
+            if basename in ignored_files:
```

Comment:
I think that is best to have the comment inside the `if` block.

But with the newer variable names, I don't think that the comment is needed anymore.

my previous request for a comment was for a different code.



```suggestion
            if basename in ignored_files:
                # The file is configured to be ignored.
```